resources/liquibase/001-initial-schema.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" logicalFilePath="001-initial-schema.xml">
+  <changeSet id="001" author="guseyn">
+    <sql>
+      CREATE TABLE "user" (
+        id SERIAL PRIMARY KEY,
+        login VARCHAR(64) NOT NULL,
```

Comment:
@Guseyn why `login` can't be primary key? Why do we need `id`?

Reply:
@yegor user can be with the same login on different platforms, I think we should add here `platform` as well, so we can put links to user profiles properly

Reply:
@Guseyn in this case it will be a different user for us. We will only work with GitHub to identify users. That's it.